Update HOOKS.md doc (#2587)
microsoft/BotFramework-WebChat: HOOKS.md

This function will return one of the following dictation states:

> -  `IDLE`: Recognition engine is idle, not recognizing

Reviewer: ```suggestion
-  `IDLE`: Recognition engine is idle; not recognizing
```

-  `WILL_START`: Will start recognition after synthesis completed

> -  `STARTING`: Recognition engine is starting, not accepting any inputs

Reviewer: ```suggestion
-  `STARTING`: Recognition engine is starting; not accepting any inputs
```

-  `DICTATING`: Recognition engine is accepting input

> -  `STOPPING`: Recognition engine is stopping, not accepting any inputs

Reviewer: ```suggestion
-  `STOPPING`: Recognition engine is stopping; not accepting any inputs
```

useGrammars(): [string[]]
```

> This function will return grammars for speech-to-text. Grammars is a list of words to suggests the speech-to-text engine to bias towards. It is commonly used for selecting the correct words with same pronounciations, e.g. Bellevue vs. Bellview vs. Bellvue.

Reviewer: ```suggestion
This function will return grammars for speech-to-text. Grammars is a list of words provided by the implementer for the speech-to-text engine to bias towards. It is commonly used for selecting the correct words with same or similar pronunciations, e.g. Bellevue vs. Bellview vs. Bellvue.
```
